packages/RawButton/test/RawButton.spec.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import RawButton from "../src";
+
+const noop = () => {};
+
+const pressEnter = { key: "Enter" };
+const pressSpace = { key: " " };
+
+function renderComponent(props = {}) {
+  const defaultProps = {
+    onClick: noop,
+  };
+
+  return render(
+    <RawButton {...defaultProps} {...props}>
+      {props.children || "happy button"}
+    </RawButton>
+  );
+}
+
+describe("RawButton", () => {
+  it("should render with default props", () => {
+    const { getByText, getByRole, container } = renderComponent();
+
+    expect(getByText(/happy button/i)).toBeInTheDocument();
+    expect(getByRole("button")).toBeInTheDocument();
+    expect(container.querySelector('[tabindex="0"]')).toBeInTheDocument();
+    expect(container.querySelector('[aria-disabled="false"]')).toBeInTheDocument();
+  });
+
+  it("should render with custom props", () => {
+    const customProps = {
+      a11yText: "include me",
+      isDisabled: true,
+      role: "listitem",
+      tabIndex: -1,
+    };
+    const { getByRole, container } = renderComponent(customProps);
+
+    expect(container.querySelector('[aria-label="include me"]')).toBeInTheDocument();
```

Comment:
Maybe can use `getByLabelText ` ? Their doc said it can find by `aria-label`

Reply:
`getByLabelText` won't find by `aria-label` by by `<label>`'s text.

Example:
```<label for="username-input">Username</label>```

Reply:
Allison appears to be correct, it _does_ select based on the `aria-label` too! Pro tip!